.github/workflows/tests.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        php: ['7.0', '7.1', '7.2', '7.3', '7.4', '8.0', '8.1']
+        php: ['7.4', '8.0', '8.1']
```

Comment:
8.0 can be dropped as well, as it's not allowed in the composer.json list.

I'd suggest to add 8.2 after tests have been fixed in general.

Reply:
I allowed myself to make the changes directly in your branch. Hope that's ok.